Automattic/woocommerce-payments
# Update express checkout max button height to 55px

Fixes #9006 

#### Changes proposed in this Pull Request




This is a small update to set the maximum button height to 55px for WooPay, Google Pay, and Apple Pay. This was the result of this [discussion](https://github.com/Automattic/woocommerce-payments/issues/8951#issuecomment-2183177946) in which it was observed the Stripe Express Checkout Element (ECE) has a max height of 55px. In the linked issue it was determined we should match Stripe's setting here of 55px. This PR makes the adjustment so that all buttons under our control will share this max height for consistency. 





#### Testing instructions





1. Enable ECE locally with `npm run wp option set _wcpay_feature_stripe_ece 1`
2. Enable WooPay and Google Pay/Apple Pay
3. From the Payments Settings page, confirm that the Button size options lists `Large (55 px)` 
4. From the Payments Settings page, confirm that changing the button size option adjusts all of the displayed buttons.
5. From the Payments Settings page, set the button size to `Large (55 px)`. Inspect all buttons displayed and confirm they are rendering at 55px height.
6. From the shop single product page, confirm the buttons display and render at 55px height.
7. From the shop single product page, confirm in browser dev tools that `wcpayConfig.woopayButton.height` and `wcpayExpressCheckoutParams.button.height` are both `55`. (Depending on some options `wcpayConfig` may need to be `wcpay_upe_config`).
8. Repeat steps 6 and 7 for:
    - Cart shortcode
    - Cart blocks
    - Checkout shortcode
    - Checkout blocks
9. Disable ECE locally with `npm run wp option set _wcpay_feature_stripe_ece 0`
10. Repeat steps 3 through 8 to confirm max heigh of 55px is still applied to Payment Request Buttons just as an additional precaution in case we need to disable ECE unexpectedly. Note that the dev tools check should be against `wcpayPaymentRequestParams.button.height` in this case.

-------------------


**Post merge**
